project/src/main/webapp/document.jsp

```diff
@@ -181,6 +185,18 @@
         xhttp.send(formData);
         return false;
       }
+
+      //Downloads current doument
+      function download() {
+        var text = firepad.getText();
+
+        var contentType = 'application/octet-stream';
+        var a = document.createElement('a');
+        var blob = new Blob([text], {'type':contentType});
+        a.href = window.URL.createObjectURL(blob);
+        a.download = '<%= document.getName() %>' + ".txt";
```

Comment:
change to corresponding language

Reply:
Will do once we figure out the code mirror issues